### ISSUE TITLE
[Terminal]: JAWS Screen Reader Cursor mode does not work inside Terminal app.

### ISSUE THREAD

**Issue by v-rpundir** [microsoft/terminal#12051]
### Windows Terminal version

1.12.3472.0

### Windows build number

10.0.22504.1010

### Other Software

**Test Environment:**
OS: Windows 11 Version Dev (OS Build 22504.1010)
App: Windows Terminal Preview
Screen Reader: JAWS 2021

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using 'Ctr+,'.
3. Activate 'Actions' tab and then Add a new actions 'Set tab color' with shortcut key 'Ctrl + Shift + c'.
4. Open JAWS 2021 Screen Reader.
5. Navigate inside Terminal window.
6. Turn on Cursor mode using 'Insert + Z' and then try to read static/non-interactive/non-tab able content. 
7. Observe the issues.

**User Experience:**
JAWS Screen Reader users are impacted here as JAWS users are not able to read static/non-interactive/non-tab able content.

**Attachments:**
[JAWS Screen Reader Cursor mode does not work inside Terminal app..zip](https://github.com/microsoft/terminal/files/7782791/JAWS.Screen.Reader.Cursor.mode.does.not.work.inside.Terminal.app.zip)

### Expected Behavior

JAWS Cursor mode should work inside Terminal app.

### Actual Behavior

JAWS Screen Reader Cursor mode does not work inside Terminal app as result JAWS users are not able to read static/non-interactive/non-tab able content.

**Observation:** 
Issue does not repro with Narrator as Scan mode work fine inside Terminal app.
NVDA Screen Reader Browse mode does not work throughout all the Windows apps. (Cursor/Scan/Browse are same mode with different name for different Screen Reader)

**Comment by akash07k** [comment]
JAWS virtual cursor only works in web apps or DOM.
Since terminal is not a web app here Virtual cursor won't work.
It's the default behaviour of jaws and messing with will lead to bad user experience

**Comment by v-rpundir** [comment]
@akash07k Ok but it is impacting for JAWS users as JAWS Screen Reader users will not able read non-interactive text on the Terminal app? 
Is this bug which only JAWS team can fix?

Thanks :):

**Comment by akash07k** [comment]
@v-rpundir  Routing JAWS Cursor to PC cursor works in this case.

**Comment by zadjii-msft** [comment]
This does feel like something that JAWS might have to fix. Curious that it doesn't mention "UWP app" for Terminal, when it does for the focus assist one. Terminal isn't a UWP, but it does use XAML and I'd imagine that JAWS would still detect the Terminal as a uwp-like app with UWP-like UIA support. 

Does this still happen on recent builds? We've gone through a few WinUI updates and the settings UI has been entirely redesigned.

**Comment by akash07k** [comment]
@zadjii-msft  JAWS works somewhat better with recent builds of WT.
I've tried it last week and it was good so far.

**Comment by v-rpundir** [comment]
@zadjii-msft @akash07k JAWS Virtual PC Cursor mode does not work my side on Terminal Preview Version: 1.15.2003.0 as result I am not able to read non-focusable item like Terminal Version number etc.
